### ISSUE TITLE
Account merge

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#1023]
It would be useful to have an account merging tool, because duplicate accounts are a real problem and I doubt we can prevent their creation entirely.

**Comment by trebmuh** [comment]
Great idea.

**Comment by Matthias84** [comment]
Yes, if users add organisations for pledgest, they seem to create dublicates pretty easily. For example:
* https://liberapay.com/on/twitter/videolan
* https://liberapay.com/on/github/videolan
* https://liberapay.com/on/twitter/supertuxkart
* https://liberapay.com/on/github/supertuxkart
* https://liberapay.com/on/twitter/netzpolitik
* https://liberapay.com/on/twitter/netzpolitik_org
* ...

I'm not familar with the LP trust modell (avoid that unofficial accounts can take money for the whole organisation), but this feature could fix the counter which is currently split accross all submitted profiles.

**Comment by deivid-rodriguez** [comment]
I run into this recently, so I added my :+1:! :)

Is this something you fix by hand per request or is it something we have to live with for the moment? I ask because this bit me when I couldn't link my Paypal account to my current account, because the email was linked to my duplicated account. I guess I could just delete the duplicate account, but it's nice to not lose your liberapay history... :slightly_smiling_face:

**Comment by Changaco** [comment]
I don't merge accounts manually, it's too complex and tricky, instead I just push aside the duplicate accounts by modifying email addresses.
